src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaCapturedVariable.cs

```diff
@@ -98,5 +97,10 @@ internal override bool IsCapturedFrame
                 return _isThis;
             }
         }
+
+        internal override bool EmitDynamicAttribute
+        {
+            get { return true; }
```

Comment:
Nitpick:  consider braces on own line for consistency or change all properties to ```=>``` syntax.